src/wasm.cc

```diff
@@ -302,36 +286,6 @@ bool WasmBase::initialize(const std::string &code, bool allow_precompiled) {
     if (!ok) {
       return false;
     }
-    auto metadata = wasm_vm_->getCustomSection("emscripten_metadata");
```

Comment:
nit: looks like we no longer need to expose getCustomSection in WasmVm class, although we may need them later?

Reply:
It's used to extract precompiled modules in V8 (which we could make internal), and in some tests in Envoy.